invadedhouse/models/house.py

```diff
@@ -117,6 +121,11 @@ def build_house(self):
             'Not so awesome. Hey, did you loot the trash can? Let\'s try the sword in there.',
             1
         )
+        knife = Weapon(
+            'Knife',
+            'Dangerous kitchen utencil.',
+            5
+        )
```

Comment:
I was originally gonna use this for the secret command, but I didn't. It is a secret weapon, though, so let's keep it